### ISSUE TITLE
base-to-new evaluation setting

### ISSUE THREAD

**Issue by tian1327** [MaxZanella/CLIP-LoRA#15]
Dear author, thank you for the good work and congrats on making into CVPRW! 

In the comparative methods section, the paper says "We compare CLIP-LoRA to several prompt-based methods: CoOp [63] (4) with 4 learn- able tokens, CoOp [63] (16) with 16 learnable tokens, Co- CoOp [62], PLOT++ [5] which is an adaption of the original PLOT proposed by the same authors specifically designed for transformer architectures, KgCoOp [53], MaPLe [26] for which `we follow the training procedure of their ”base- to-new” setting`,". 

Does this mean the Table 2 and Fig.2 report the test accuracy on the new classes only? Or the base classes? Why don't include the base performance?

**Comment by MaxZanella** [comment]
Hello,

Thank you for your interest in our work! To clarify, our benchmarks focus specifically on the few-shot setting (where all the classes are provided in the labeled examples). However, exploring the base-to-new setting with LoRA could indeed be an interesting direction for future work! Regarding the paragraph in question, we were referring to the choice of hyperparameters for KgCoOp and MaPLe (such as the number of epochs). Just to confirm, the reported test accuracy in Figure 2 and Table 2 is calculated across all classes.

I hope this clears things up!

**Comment by tian1327** [comment]
Thank you very much for the clarification! I am looking forward to seeing better work from you in the future!
